rfcs/formats/network/iin_method.md

```diff
@@ -0,0 +1,264 @@
+
+See  [Network DID Identity format](../../formats/identity.md) for details about Network DID format.
+
+
+##### Preparing Network DID creation request
+
+Creating a Network DID is only possible by the DID's controller. Therefore to create a Network DID for the first time, the request has to be authenticated by verifying attestations of all participants of the network. This is done while forming the Network DID creation request.
+
+A Network DID creation request has a `NetworkDIDDocument` and `signatures`. `NetworkDIDDocument` contains the Network DID Document with the `networkParticipants` and a `GroupMultiSig` verificationMethod. The `signatures` is a set of signatures from all participants.
+
+> Note: During NetworkDID creation, the `updatePolicy` of the `GroupMultiSig` verificationMethod is not used. Instead. the request has to be authenticated against all the participants in `networkParticipants` list.
+
+Example NetworkDID creation request:
+
+```json
+{
+  "NetworkDIDDocument": {
+    "id": "did:iin:<iin_name>:<network_name>",
+    "networkParticipants": [
+      "did:iin:<iin_name>:<network_participant_1>",
+      "did:iin:<iin_name>:<network_participant_2>",
+      "did:iin:<iin_name>:<network_participant_3>"
+    ],
+    "verificationMethod": [{
+        "id": "did:iin:<iin_name>:<network_name>#multisig",
+        "type": "GroupMultiSig",
+        "controller": "did:iin:<iin_name>:<network_name>",
+        "multisigKeys": [
+          "did:iin:<iin_name>:<network_participant_1>#key1",
+          "did:iin:<iin_name>:<network_participant_2>#key3",
+          "did:iin:<iin_name>:<network_participant_3>#key1"
+        ],
+        "updatePolicy": {
+          "id": "did:iin:<iin_name>:<network_name>#updatepolicy",
+          "controller": "did:iin:<iin_name>:<network_name>",
+          "type": "VerifiableCondition2021",
+          "conditionAnd": [{
+              "id": "did:iin:<iin_name>:<network_name>#updatepolicy-1",
+              "controller": "did:iin:<iin_name>:<network_name>",
+              "type": "VerifiableCondition2021",
+              "conditionOr": ["did:iin:<iin_name>:<network_participant_3>#key1",
+                "did:iin:<iin_name>:<network_participant_2>#key3"
+              ]
+            },
+            "did:iin:<iin_name>:<network_participant_1>#key1"
+          ]
+        }
+      },
+
+      {
+        "id": "did:iin:<iin_name>:<network_name>#fabriccerts",
+        "type": "DataplaneCredentials",
+        "controller": "did:iin:<iin_name>:<network_name>",
+        "FabricCredentials": {
+          "did:iin:<iin_name>:<network_participant_1>": "Certificate3_Hash",
+          "did:iin:<iin_name>:<network_participant_2>": "Certificate2_Hash",
+          "did:iin:<iin_name>:<network_participant_3>": "Certificate3_Hash"
+        }
+      }
+    ],
+    "authentication": [
+      "did:iin:<iin_name>:<network_name>#multisig"
+    ],
+    "relayEndpoints": [{
```

Comment:
Change this to `networkGatewayEndpoints`. And below the example, mention what they represent. You can link to the Weaver relay RFC there if you wish. (The term `relayEndpoints` is not well-known and may be confusing to people.)